### PR TITLE
4 current location

### LINKED ISSUE #4: Specify current location
As a user
I would like to specify my current location
So that I can meet up with other coders

### PR DESCRIPTION
**Pull request by Jojograndjojo**
closes #4

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,3 +1,9 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+  before_action :load_location_manager
+
+  def load_location_manager
+      @location_manager = LocationManager
+  end
+
 end
```
Comment (chdezmar) [inline app/controllers/application_controller.rb]: nice

File: test/controllers/location_managers_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class LocationManagersControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
```
Comment (chdezmar) [inline test/controllers/location_managers_controller_test.rb]: ?

File: spec/features/user_sign_up_spec.rb [hunk 3]
```diff
@@ -9,10 +9,13 @@
       expect(page).to have_link('Sign in')
       end
     end
+
   context 'has signed up' do
     it 'should see a sign out link' do
```
Comment (Jojograndjojo) [inline spec/features/user_sign_up_spec.rb]: may we change that to, 'it should feel a sign out link' I fid that much more organic

File: test/models/location_manager_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class LocationManagerTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```
Comment (chdezmar) [inline test/models/location_manager_test.rb]: These test files are not necessary

File: spec/features/check_in_spec.rb [hunk 5]
```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+
+feature 'User' do
+  context 'has not signed up yet' do
+    xit 'should not be able to check in a place' do
```
Comment (Rb2030) [inline spec/features/check_in_spec.rb]: What's with the x guys? Either it's a test or not. Hassan and I both expect better than this.

File: spec/features/check_in_spec.rb [hunk 6]
```diff
@@ -1,23 +1,39 @@
 require 'rails_helper'
 
 
-feature 'User' do
+feature 'Check in' do
   context 'has not signed up yet' do
-    xit 'should not be able to check in a place' do
```
Comment (Rb2030) [inline spec/features/check_in_spec.rb]: Another x.

**Comment by Jojograndjojo** [comment]
:neckbeard:

**Comment by Rb2030** [comment]
Nice work guys generally. Room for improvement, but I understand the time restraints you are all under. :shipit:

**Comment by toby676** [comment]
:shipit:

**Pull request merged**